Jenkinsfile

```diff
@@ -96,6 +96,17 @@ pipeline {
                       }
                   }
                 }
+                stage('AWS Cloud image (smokes)') {
+                  agent { label 'linux' }
+                  steps {
+                      sh 'make -C distribution aws-cloud-container-check'
+                  }
+                  post {
+                      always {
+                          archiveArtifacts artifacts: '**/build/tests-run*/**.log*'
```

Comment:
BTW this is a less than friendly way of retaining logs. Better to either inline into the build log; or (if possible) convert to JUnit format and publish properly in Jenkins.